### ISSUE TITLE
storybook-vike

### ISSUE THREAD

**Issue by Kenzo-Wada** [vikejs/vike#1595]
### Description

Hi @brillout 

Thanks a lot for maintaining such a awesome Framework!

WDYT about creating a integration plugin for storybook named like `storybook-vike` so that we can use APIs of vike inside stories file.
Im imagining something like Vike version of `@storybook/nextjs` .

**Comment by brillout** [comment]
Hi Kenzo, I'm glad you're enjoying Vike!

Sounds good :eyes: Would it require changes from Vike core?

**Comment by Kenzo-Wada** [comment]
> Would it require changes from Vike core?

I don't think it is necessary, and if it is needed in the future, I believe that storybook-vike should address Vike's destructive changes, not modify Vike!

P.S. I didn't know where to ask for advice, so I asked here, sorry if I'm in the wrong place.

**Comment by brillout** [comment]
:+1: Sounds good, we can give it a try and see. (My only concern would be complex Vike core changes that would slow down other high priority issues. We can try and see how it goes.)

No worries, moving this to GitHub Discussions.
